packages/cozy-client/src/utils.js

```diff
@@ -39,7 +39,11 @@ export const isQueryLoading = col => {
     console.warn('isQueryLoading called on falsy value.') // eslint-disable-line no-console
     return false
   }
-  return col.fetchStatus === 'loading' || col.fetchStatus === 'pending'
+  return (
+    col.fetchStatus === 'loading' ||
+    col.fetchStatus === 'pending' ||
+    col.isFetching
```

Comment:
I need to have a deeper look on the CozyClient code, to find if there is other place like this, what need a change.

Reply:
This is exactly what I don't want to see in Cozy-Client. Your query has data loaded, you should display them. `isFetching` should only be there to add a little information on the UI side. This is a different concept.